content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -153,7 +153,7 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
 1. Test to ensure the version you installed is up-to-date:
 
    ```bash
-   kubectl version --client
+   kubectl version
```

Comment:
This change is not helpful. We want to avoid trying to query a Kubernetes cluster.